tests/neo-vm.Tests/Extensions/ByteArrayExtensions.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Threading;
+using Neo.Test.Cryptography;
```

Comment:
We can remove this `Neo.Test.Cryptography` include, since it looks like only RIPEMD160Managed is included, right?
I checked the implementation of RIPEMD160Managed and it is equivalent to the standard one in System.Security.Cryptography. So, we can use standard, the same way we do for SHA256.

Looks like ECC code pushed here is on Neo.Cryptography.ECC namespace, which is also not imported anywhere. So it can also be safely removed.

Reply:
Just found it... you recreated Crypto lib too...

Ok then. Let's approve this first, then we decide how to deal with this.

Reply:
https://docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.ripemd160managed?view=netframework-4.7.2 is not included in dotnet core :(

Reply:
I didn't know that... ok, one more reason to leave that for now. In the future we come back to this discussion.